tests/Microsoft.Identity.Test.Unit/WwwAuthenticateParametersTests.cs

```diff
@@ -292,5 +379,17 @@ private static HttpResponseMessage CreateInvalidTokenHttpErrorResponse(string te
                 }
             };
         }
+
+        private static HttpResponseMessage CreateBearerAndPopHttpResponse()
+        {
+            return new HttpResponseMessage(HttpStatusCode.Unauthorized)
+            {
+                Headers =
+                {
+                    { WwwAuthenticateHeaderName, $"Bearer authorization_uri=\"https://login.microsoftonline.com/TenantId/oauth2/authorize\", resource_id=\"https://endpoint/\"" },

```

Comment:
The parser does not cover the case of `// Challenges specified in single header
WWW-Authenticate: challenge1, ..., challengeN`. This is probably ok, but an explicit error needs to be added.

Reply:
This scenario is covered by the .Net Http Parsing logic. However, it work hurt t add the test case to be 100% sure.